src/include/catalog/pg_auth_time_constraint.h

```diff
@@ -71,4 +72,11 @@ typedef FormData_pg_auth_time_constraint *Form_pg_auth_time_constraint;
 #define Anum_pg_auth_time_constraint_end_day	4
 #define Anum_pg_auth_time_constraint_end_time	5
 
+#define Schema_pg_auth_time_constraint \
+{6070, {"authid"}    ,   26, -1, 0, 4, 1, 0, -1, -1, true           , 'p' ,'i', true, false, false, true, 0, { 0 } }, \
+{6070, {"start_day"} ,   21, -1, 0, 2, 2, 0, -1, -1, true           , 'p' ,'s', true, false, false, true, 0, { 0 } }, \
+{6070, {"start_time"}, 1083, -1, 0, 8, 3, 0, -1, -1, FLOAT8PASSBYVAL, 'p' ,'d', true, false, false, true, 0, { 0 } }, \
+{6070, {"end_day"}   ,   21, -1, 0, 2, 4, 0, -1, -1, true           , 'p' ,'s', true, false, false, true, 0, { 0 } }, \
+{6070, {"end_time"}  , 1083, -1, 0, 8, 5, 0, -1, -1, FLOAT8PASSBYVAL, 'p' ,'d', true, false, false, true, 0, { 0 } }
+
```

Comment:
Review especially appreciated over here; this table is mostly cobbled together from other schema examples I found. I supposed it will go away later in the merge with `BKI_SCHEMA_MACRO`.

Reply:
LGTM. Yeah, we will get rid of these later on, good riddance.